app/src/main/java/chat/rocket/android/chatroom/adapter/MessageViewHolder.kt

```diff
@@ -1,21 +1,44 @@
 package chat.rocket.android.chatroom.adapter
 
 import android.graphics.Color
+import android.graphics.drawable.Drawable
+import android.text.Spannable
+import android.text.Spanned
 import android.text.method.LinkMovementMethod
+import android.text.style.ImageSpan
 import android.view.View
 import androidx.core.view.isVisible
+import androidx.core.view.postDelayed
 import chat.rocket.android.R
 import chat.rocket.android.chatroom.uimodel.MessageUiModel
 import chat.rocket.android.emoji.EmojiReactionListener
 import chat.rocket.core.model.isSystemMessage
+import com.bumptech.glide.load.resource.gif.GifDrawable
 import kotlinx.android.synthetic.main.avatar.view.*
 import kotlinx.android.synthetic.main.item_message.view.*
 
 class MessageViewHolder(
     itemView: View,
     listener: ActionsListener,
     reactionListener: EmojiReactionListener? = null
-) : BaseViewHolder<MessageUiModel>(itemView, listener, reactionListener) {
+) : BaseViewHolder<MessageUiModel>(itemView, listener, reactionListener), Drawable.Callback {
+    override fun unscheduleDrawable(who: Drawable?, what: Runnable?) {
+        with(itemView) {
```

Comment:
Not needed to use `with(...)` here because there is only one block being used.

Reply:
If I remove with here I'm almost sure the ide will not recognize my android extensions' imports.

Reply:
You can use `itemView.text_content.removeCallbacks(what)`instead.

Reply:
OK, I personally don't like this approach because it will make me repeat `itemView.` all over, but I did the changes anyway :+1: